tests/scripts/fast-reboot

```diff
@@ -263,7 +263,7 @@ function reboot_pre_check()
     fi
     
     # Make sure ASIC configuration has not changed between images
-    ASIC_CONFIG_CHECK_SCRIPT="/usr/bin/asic_config_check"
+    ASIC_CONFIG_CHECK_SCRIPT="/usr/local/bin/asic_config_check"
```

Comment:
i do not think i can back port the image change into 201911 branch. can you maintain back compatible for the test?

Reply:
Added backward-compatibility in latest commit.